### PR TITLE
Move gRPC participant to SDK repo

### PR DESCRIPTION
**Pull request by Robert-Steiner**
Moves the gRPC participant of the [xain](https://github.com/xainag/xain) repository into the new [xain-sdk](https://github.com/xainag/xain-sdk) repository.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+root = true
```
Comment (r-marques) [inline .editorconfig]: Is this file really needed. I don't even know what uses this

Comment (r-marques) [inline .editorconfig]: Is this file really needed. I don't even know what uses this

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,141 @@
+# Project specific files and directories
```
Comment (r-marques) [inline .gitignore]: github provides a collection of .gitignore files for multiple languages. Since this is a new repo I would probably start with that https://github.com/github/gitignore/blob/master/Python.gitignore.

Also I'm afraid that having pants related stuff there will cause developers nightmares :p

File: RELEASE_PROCESS.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+A release on git is just a tagged commit on the `master` branch.
+
+### How to do a Github Release
+
+Here we detail the process of creating a new Github release.
+
+1. Create and merge a pull request that:
+    - increases the version number in
+      [`xain/__version__.py`](https://github.com/xainag/xain/blob/master/xain/__version__.py)
```
Comment (r-marques) [inline RELEASE_PROCESS.md]: Links on this release process will need to be updated

File: protobuf/xain/grpc/coordinator.proto [hunk 4]
```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
```
Comment (r-marques) [inline protobuf/xain/grpc/coordinator.proto]: For now we are going with the option of duplicating the proto file?

File: conftest.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
+"""
```
Comment (r-marques) [inline conftest.py]: If we have a `tests/` folder we probably don't want to have `conftest.py` outside the test folder.

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+import sys
+
+from setuptools import find_packages, setup
+from setuptools.command.develop import develop
+
+if sys.version_info < (3, 6):
+    sys.exit("Please use Python version 3.6 or higher.")
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+version_file_path = os.path.join(project_dir, "xain/__version__.py")
```
Comment (r-marques) [inline setup.py]: are we keeping the name `xain` for the source folder?

File: .editorconfig (hunk 1 above)
Comment (Robert-Steiner) [reply .editorconfig]: Right, I have removed it.

File: .gitignore (hunk 2 above)
Comment (Robert-Steiner) [reply .gitignore]: I saw that too :D Maybe we should remove it in the xain repository as well.

File: protobuf/xain/grpc/coordinator.proto (hunk 4 above)
Comment (Robert-Steiner) [reply protobuf/xain/grpc/coordinator.proto]: For now yes. I will create a ticket to move the proto files into a separate repository (for xain and xain-sdk).
